harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
you're right, i think i'd misunderstood the comment. i guess i can just rearrange the descriptor checks, but it still seems like a footgun. side effects ◉︵◉